deepbox/src/main/java/ch/cyberduck/core/deepbox/DeepboxDirectoryFeature.java

```diff
@@ -52,6 +53,9 @@ public DeepboxDirectoryFeature(final DeepboxSession session, final DeepboxIdProv
     @Override
     public Path mkdir(final Path folder, final TransferStatus status) throws BackgroundException {
         try {
+            if(new DeepboxFindFeature(session, fileid).find(folder)) {
```

Comment:
Use `status.isExists()` and trash folder when existing. Refer to `DriveMoveFeature#move`.

Reply:
Copied from `DriveDirectoryFeature`.

Reply:
`CreateDirectoryWorker` and `TouchWorker` do not determine if a target might already exist. Thus we cannot rely on the `exists` flag in this case but expect implementations to throw failure on conflict. This was introduced in #14168.